include/seqan3/core/range/detail/inherited_iterator_base.hpp

```diff
@@ -109,18 +109,22 @@ class inherited_iterator_base : public std::conditional_t<std::is_pointer_v<base
         : member{std::move(it)}
     {}
     //!\}
+    //
 
-    //!\brief Get a copy of the base.
-    constexpr base_t base() const &
```

Comment:
So now you cannot return a copy anymore?

Reply:
Copying should still work. It is just not a forced copy any more.